src/utils/setup/remove-page-if-staged.js

```diff
@@ -0,0 +1,10 @@
+const removePageIfStaged = (page, deletePage, stagingPages) => {
+    if (
+        process.env.SNOOTY_ENV === 'production' &&
+        stagingPages.includes(page.path)
+    ) {
+        deletePage(page);
```

Comment:
I'm assuming this also won't show on tagmanager since the page is removed before the end of this lifecycle step but not sure how to verify locally